cluster/gce/trusty/configure-helper.sh

```diff
@@ -726,7 +726,7 @@ Note: This looks like a development version, which might not be present on GitHu
 If it isn't, the closest tag is at:
   https://github.com/kubernetes/kubernetes/tree/${gitref}
 "
-    gitref="${version//*+/}"
+    gitref=$(echo ${version} | sed -e 's/.*+//')
```

Comment:
The line "gitref="${version//*+/}"" does not work in upstart job, but works in shell script. The GCI config executes a shell script via systemd, but trusty config directly calls shell functions in upstart jobs so it does not work.

This is a problem I once mentioned in the discussion of maintenance plan for trusty code. We decided not to guarantee it green in the master branch to reduce the maintenance cost. As a result, there have been several key fixes merged in GCI config are not made for trusty config, and trusty config is broken. The downside is that it is hard to verify if a change is 100% correct. Here is one case of such situation. I noticed that gitref="${version//*+/}" does not work, but apparently I did not make a change in the master branch or release-1.3. And later in 1.2 branch I saw the failure and so I made this PR to ensure it is green in release-1.2 branch.